simple-chat/Podfile.lock

```diff
@@ -1,15 +1,15 @@
 PODS:
-  - JSQMessagesViewController (7.3.4):
+  - JSQMessagesViewController (7.3.5):
     - JSQSystemSoundPlayer (~> 2.0.1)
   - JSQSystemSoundPlayer (2.0.1)
 
 DEPENDENCIES:
   - JSQMessagesViewController
 
 SPEC CHECKSUMS:
-  JSQMessagesViewController: 39fed975e3c9f8eba7292071e29eeb541d105e66
+  JSQMessagesViewController: 6587f56aedbcb150a8ba1d968e1ef58b1bc758b7
   JSQSystemSoundPlayer: c5850e77a4363ffd374cd851154b9af93264ed8d
 
 PODFILE CHECKSUM: 7d1ef276401e740a2993e6e057ce81cb7928e4af
 
-COCOAPODS: 1.2.0
+COCOAPODS: 1.3.1
```

Comment:
A bunch of the png files are showing up as deleted and readded in this PR, when it appears that there is actually no change to the file.  If it is true, could we rework the commits so that these are excluded.

Reply:
Unfortunately, those files are introduced by a dependency that is managed by Cocoapods. I'm not sure what has changed, but I think it would be good to include them.